masschallenge/impact-api
# AC-5067: Fix 500 for GET of /v1/user/35908 (or any user without a profile)

Work done:
- Code was fixed to work with users that have no profile objects.

To test:
- `make load-remote-db DB_FILE_NAME=initial_schema.sql.gz`
- Grant the appropriate permissions to a super user account.  E.g., `make grant-permissions PERMISSION_USER=13514-user@example.com PERMISSION_CLASSES=v1_clients`
- Login as that user.  E.g., 13514-user@example.com
- Visit http://localhost:8000/api/v1/user/35908/
- On development it should generate a 500 and on the branch it should be fine.
- Try other calls with this user or with any of the other "broken" users (39658, 38978, 41341, and 44399)
- Note: Some PATCH calls will now return 403s with appropriate error messages if you try to PATCH a field that doesn't exist.  You should not be able to generate any 500s.
